src/OrchardCore.Modules/OrchardCore.Apis.GraphQL/Services/SerialDocumentExecuter.cs

```diff
@@ -0,0 +1,36 @@
+using GraphQL;
+using GraphQL.Execution;
+using GraphQL.Language.AST;
+using System;
+
+namespace OrchardCore.Apis.GraphQL.Services
+{
+    /// <summary>
+    /// Defines a custom execution strategy for queries such that async lambdas are not executed in parallel.
+    /// c.f. https://github.com/OrchardCMS/OrchardCore/issues/3029
+    /// </summary>
+    public class SerialDocumentExecuter : DocumentExecuter
+    {
+        private static IExecutionStrategy ParallelExecutionStrategy = new ParallelExecutionStrategy();
```

Comment:
Unused field?

Reply:
Correct, just so that we know it exists. The actual implementation in graphql uses it, and this is what the issue is about. I prefer to leave it then.

You can say a comment would be better :/

Reply:
> You can say a comment would be better :/

I dare say it, yeah :smile:

Reply:
thought the same thing earlier when looking into another issue and looking at it @PinpointTownes ;)